### ISSUE TITLE
title for techniques details page

### ISSUE THREAD

**Issue by nitedog** [w3c/wai-axsdb-web#20]
Page: "details for technique h44 ..."
Issue: make "Creating a logical tab order" a sub-title ("Details for Technique H44" should be the prominent heading)
